### ISSUE TITLE
It is not possible to add MS tile image meta via default_head_blocks.xml

### ISSUE THREAD

**Issue by chicgeek** [magento/magento2#5023]
It is currently not possible to add a `msapplication-TileImage` meta value to the head as the image path is not processed. This meta uses the `content` attribute instead of `src` or `href` and is used to specify the path for the tile images for Windows devices. Other similar icon paths that use `src` have the paths resolved correctly.

This inability to support this meta tag has been evident since before the GA release and is still an issue in 2.0.6.
## Preconditions
Magento 2.0.6
## Steps to reproduce

Add the following to `default_head_blocks.xml`. I have added other entries to this example to this show how other link / meta tags are supported, though the last line (`msapplication-TileImage`) is all that is required to see the broken behaviour.

``` xml
<link rel="icon" type="image/png" src="images/favicons/favicon-16x16.png" sizes="16x16"/>
<link rel="manifest" src="images/favicons/default/manifest.json"/>
<meta name="theme-color" content="#00989e"/>
<meta name="msapplication-TileColor" content="#00989e"/>
<meta name="msapplication-TileImage" content="images/favicons/default/mstile-144x144.png"/>
```
## Expected result

The `content` attribute for `msapplication-TileImage` should resolve to a localised theme path. For example:

``` xml
<meta name="msapplication-TileImage" content="http://example.com/static/frontend/vendor/theme/en_GB/images/favicons/default/mstile-144x144.png">
```
## Actual result

The `content` attribute for `msapplication-TileImage` does not resolve to a localised theme path. 

``` xml
<meta name="msapplication-TileImage" content="images/favicons/default/mstile-144x144.png">
```

**Comment by korostii** [comment]
For anyone struggling with this issue: keep in mind that, as a workaround, one can still add a block (even a simple \Magento\Framework\View\Element\Template will do the trick) into 
`head.additional` and then inside it generate the full URL using `$block->getViewFileUrl()`.

**Comment by magento-engcom-team** [comment]
@chicgeek, thank you for your report.
We've created internal ticket(s) MAGETWO-82524 to track progress on the issue.

**Comment by therouv** [comment]
#mhlej18

**Comment by magento-engcom-team** [comment]
@therouv thank you for joining. Please accept team invitation [here](https://github.com/orgs/magento/invitation) and self-assign the issue.

**Comment by magento-engcom-team** [comment]
Hi @chicgeek. Thank you for your report. 
The issue has been fixed in magento/magento2#21798 by @vasilii-b in 2.3-develop branch
Related commit(s):
 - [a2551afc1cd3e8f43a6c9e05d9aa9f3939bcfda5](https://github.com/magento/magento2/commit/a2551afc1cd3e8f43a6c9e05d9aa9f3939bcfda5)
 - [3d193bfade9019a08f162e9bf9b30c625dc1b098](https://github.com/magento/magento2/commit/3d193bfade9019a08f162e9bf9b30c625dc1b098)
 - [2faa1ecc3ec8e69752070c31239385114d4941c4](https://github.com/magento/magento2/commit/2faa1ecc3ec8e69752070c31239385114d4941c4)
 - [56a368fe74ca8f058a5f38b2b0b900d247e34724](https://github.com/magento/magento2/commit/56a368fe74ca8f058a5f38b2b0b900d247e34724)
 - [b488bd4f1f46f860c175fd75ec0d0b3a1010afa2](https://github.com/magento/magento2/commit/b488bd4f1f46f860c175fd75ec0d0b3a1010afa2)
 - [8892b0bf7275426f4aad603252d0e40ab39af675](https://github.com/magento/magento2/commit/8892b0bf7275426f4aad603252d0e40ab39af675)
 - [a917ee7ee60b891b785dbdced8ee366a6de6c934](https://github.com/magento/magento2/commit/a917ee7ee60b891b785dbdced8ee366a6de6c934)

The fix will be available with the upcoming 2.3.4 release.
